src/components/Pages/Topic.js

```diff
@@ -11,23 +11,27 @@ import SectionHeader from 'components/SectionHeader';
 import TileGroup from 'components/Tiles/TileGroup';
 
 const Topic = ({
-  topic: { theme, text: title, description, callToAction, tiles },
+  topic: {
+    theme,
+    text: title,
+    description,
+    callToAction,
```

Comment:
Think I saw you remove callToAction from Joplin, no harm having it in there for now and removing later